src/steamship/agents/examples/my_assistant.py

```diff
@@ -59,11 +62,13 @@ def sync_emit(blocks: List[Block], meta: Metadata):
 
         context.emit_funcs.append(sync_emit)
         self.run_agent(self._agent, context)
+        # TODO: is this right?
```

Comment:
let's remove this TODO and maybe add something like:

`#  Without appending the most recent message, the agent service will not be able to plan/respond appropriately.`

Or something more explanatory like that.